style.css

```diff
@@ -0,0 +1,74 @@
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 16px 24px;
+}
+
+.logo {
+  color: #6070ff;
+  font-family: 'Poppins', sans-serif;
+  font-weight: 700;
+  font-size: 18px;
+  line-height: 20px;
+}
+
+.nav-icon {
+  display: flex;
+  flex-direction: column;
+  gap: 4px;
+}
+
+.icon {
+  display: inline;
+  background-color: #6070ff;
+  width: 18px;
+  height: 2px;
+  border-radius: 2px;
+}
+
+main {
+  display: flex;
+  flex-direction: column;
+  background: url(/background.svg) no-repeat;

```

Comment:
- [ ] [REQUIRED] As requested ♻️ in project requirements 🧾 This section should have a proper `background-image` (I think you should use `./background.svg` in path instead of using the `/background.svg`).